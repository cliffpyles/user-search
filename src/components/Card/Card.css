.Card {
  padding: 1rem;
  border: var(--Card--Border, 1px solid #e1e1e1);
  background: var(--Card--Background, #fff);
  color: var(--Card--Color, #333);
  font-family: var(--Card--FontFamily, system-ui);
  font-size: var(--Card--FontSize, 16px);
}
