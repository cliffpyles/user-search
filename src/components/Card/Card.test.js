import React from 'react'
import Renderer from 'react-test-renderer'
import Card from './'

test('can render a Card', () => {
  const component = Renderer.create(
    <Card>
      <p>Here goes the card!</p>
    </Card>
  )

  expect(component.toJSON()).toMatchSnapshot()
})
