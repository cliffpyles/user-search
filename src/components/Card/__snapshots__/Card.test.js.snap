// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can render a Card 1`] = `
<div
  className="Card"
>
  <p>
    Here goes the card!
  </p>
</div>
`;
